salt/utils/cloud.py

```diff
@@ -2763,15 +2780,25 @@ def list_cache_nodes_full(opts=None, provider=None, base=None):
     return minions
 
 
-def cache_nodes_ip(opts, base=None):
+def get_cache_node(minion_id, base=None):
```

Comment:
Is there a reason you're changing the name of this function? It is unclear to me why these changes are necessary and we can't move the names of functions like this without putting them on a deprecation path.